tests/setup.go

```
@@ -3,67 +3,45 @@
-
-	req := testcontainers.ContainerRequest{
-		Image:        "postgres:15",
-		ExposedPorts: []string{"5432/tcp"},
-		Env: map[string]string{
-			"POSTGRES_USER":     "test",
-			"POSTGRES_PASSWORD": "test",
-			"POSTGRES_DB":       "testdb",
-		},
-		WaitingFor: wait.ForLog("database system is ready to accept connections").WithStartupTimeout(60 * time.Second),
-	}
+	pgContainer, err := postgres.RunContainer(ctx,
```

golangci-lint

SA1019: postgres.RunContainer is deprecated: use Run instead RunContainer creates an instance of the Postgres container type 

(staticcheck)